roles/common/templates/logrotate_rules.j2

```diff
@@ -0,0 +1,43 @@
+# {{ ansible_managed }}
+# Logrotate configuration for {{ item.name }}
+# Generated by Ansible - Standardized configuration
+
+{% for path in item.paths %}
+{{ path }}
+{% endfor %} {
+    # Standardized rotation frequency - ALWAYS daily
+    daily
+    
+    # Standardized retention - configurable but with sensible default
+    rotate {{ item.options.rotate | default(logrotate_global_defaults.rotate) | default(7) }}
+    
+    # ALWAYS enforce maximum size
+    maxsize {{ item.options.maxsize | default(logrotate_global_defaults.maxsize) | default('2000M') }}
+    
+    # Standardized compression settings
+    compress
+    delaycompress
+    
+    # Standardized error handling
+    missingok
+    notifempty
+    
+    # Standardized file creation (always create new files with consistent permissions)
+    create {{ item.options.create_mode | default(logrotate_global_defaults.create_mode) | default('0644') }} {{ item.options.create_owner | default(logrotate_global_defaults.create_owner) | default('root') }} {{ item.options.create_group | default(logrotate_global_defaults.create_group) | default('root') }}
+    
+    # Use date extension for clarity so logs will say application.log-20250623-1719158400
+    dateext
+    dateformat -%Y%m%d-%s
+{% if item.options.sharedscripts is defined and item.options.sharedscripts %}
+    
+    # Shared scripts (only when explicitly needed)
+    sharedscripts
+{% endif %}
+{% if item.options.postrotate is defined and item.options.postrotate != '' %}
+    
+    # Post-rotation commands
+    postrotate
```

Comment:
This probably needs to be inside the `if` statement, so we don't end up with stray `postrotate` lines in rotation files that don't need them.